code/game/jobs/job/supervisor.dm

```diff
@@ -191,7 +191,8 @@ var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
 			            access_theatre, access_chapel_office, access_library, access_research, access_mining, access_heads_vault, access_mining_station,
 			            access_clown, access_mime, access_hop, access_RC_announce, access_keycard_auth, access_gateway, access_magistrate)
 	minimal_access = list(access_security, access_sec_doors, access_brig, access_court, access_maint_tunnels, access_lawyer, access_magistrate)
-
+	alt_titles = list("Chief Justice")
```

Comment:
Will cause confusion, the SOP has them listed under a different name. This will end extremely badly.